### PR TITLE
Create markup for cardslist

### PR DESCRIPTION
**Pull request by Pedrael**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,16 +1,64 @@
-import { Container, CssBaseline } from '@mui/material'
+import { Box, Container, CssBaseline } from '@mui/material'
 import { CustomisedAppBar } from './components/CustomisedAppBar'
 import FaceScreen from './components/FaceScreen'
+import { CardsList } from './components/CardsList'
+import { backgroundColor } from './vars'
```
Comment (ReidenXerx) [inline src/App.tsx]: Conventional name for similar files with constants is “constants “

File: src/App.tsx [hunk 2]
```diff
@@ -1,16 +1,64 @@
...
 
 function App() {
   return (
-    <>
+    <Box sx={{ backgroundColor: { backgroundColor } }}>
       <CssBaseline />
       <Container>
```
Comment (ReidenXerx) [inline src/App.tsx]: Why container and not box or stack?

File: src/components/Card.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import { Avatar, Stack } from '@mui/material'
+import { CustomisedTypography } from './CustomisedTypography'
+import { fontFamily } from '../vars'
```
Comment (ReidenXerx) [inline src/components/Card.tsx]: Same

File: src/components/CardsList.tsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+      <Box
+        display="flex"
+        flexWrap="wrap"
+        gap="29px"
+        justifyContent="center"
+        mb="50px"
+      >
+        {cards.map((item, index) => (
+          <Card
+            key={index}
```
Comment (ReidenXerx) [inline src/components/CardsList.tsx]: Awful practice
We need to use uuid for prevent react wrong indexing

File: src/components/CardsList.tsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+        Working with GET request
+      </CustomisedTypography>
+      <Box
+        display="flex"
+        flexWrap="wrap"
+        gap="29px"
+        justifyContent="center"
+        mb="50px"
+      >
+        {cards.map((item, index) => (
```
Comment (ReidenXerx) [inline src/components/CardsList.tsx]: Destructurization of item

File: src/components/FaceScreen.tsx [hunk 6]
```diff
@@ -58,7 +59,7 @@ const FaceScreen: React.FC<FaceScreenProps> = () => {
           thinking as they'll be building web interfaces with accessibility in
           mind. They should also be excited to learn, as the world of Front-End
           Development keeps evolving.
-        </Typography>
+        </CustomisedTypography>
```
Comment (ReidenXerx) [inline src/components/FaceScreen.tsx]: Better to use sense full naming for custom components

**Review by ReidenXerx — APPROVED** [review]

File: src/App.tsx (hunk 1 above)
Comment (Pedrael) [reply src/App.tsx]: fixed

File: src/App.tsx (hunk 2 above)
Comment (Pedrael) [reply src/App.tsx]: fixed

File: src/components/Card.tsx (hunk 3 above)
Comment (Pedrael) [reply src/components/Card.tsx]: fixed

File: src/components/CardsList.tsx (hunk 4 above)
Comment (Pedrael) [reply src/components/CardsList.tsx]: fixed
`{cards.map(({ title, description, email, phone }) => (
          <Card
            key={uuidv4()}
            title={title}
            description={description}
            email={email}
            phone={phone}
          />
        ))}`

File: src/components/CardsList.tsx (hunk 5 above)
Comment (Pedrael) [reply src/components/CardsList.tsx]: fixed

File: src/components/FaceScreen.tsx (hunk 6 above)
Comment (Pedrael) [reply src/components/FaceScreen.tsx]: fixed

File: src/components/CardsList.tsx [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+      <Box
+        display="flex"
+        flexWrap="wrap"
+        gap="29px"
+        justifyContent="center"
+        mb="50px"
+      >
+        {cards.map(({ title, description, email, phone }) => (
+          <Card
+            key={uuidv4()}
```
Comment (ReidenXerx) [inline src/components/CardsList.tsx]: Its better to store it in entity and extend interface

File: src/theme.ts [hunk 8]
```diff
@@ -1,45 +1,9 @@
...
-  secondaryColor,
-  backgroundColor,
-} from './vars.ts'
+import { fontFamily } from './contstants.ts'
 
 export const theme = createTheme({
   typography: {
     fontFamily: fontFamily,
   },
-  palette: {
```
Comment (ReidenXerx) [inline src/theme.ts:15]: Why did u delete it

**Review by ReidenXerx — CHANGES_REQUESTED** [review]

File: src/theme.ts (hunk 8 above)
Comment (Pedrael) [reply src/theme.ts:15]: because it is not needed here

File: src/components/CardsList.tsx (hunk 7 above)
Comment (Pedrael) [reply src/components/CardsList.tsx]: fixed

**Review by ReidenXerx — APPROVED** [review]

**Pull request merged**